README.md

```diff
@@ -164,6 +141,15 @@ To SSH into the running WordPress instance:
 ssh $(terraform output ssh_user)@$(terraform output public_ip)
 ```
 
+## Build container
+
+Part of the build runs in [a custom container](https://hub.docker.com/r/18fgsa/devsecops-builder/). To update it:
+
+```sh
+docker build --pull -t 18fgsa/devsecops-builder:alpine - < Dockerfile
+docker push 18fgsa/devsecops-builder:alpine
```

Comment:
The GSA GitHub org hasn't authorized Docker Hub, so couldn't set this up as an automated build. The sticking point to doing so will likely be that Docker Hub asks for full access to private repositories (https://github.com/docker/hub-feedback/issues/260), and the read-only option doesn't work for organizations (https://github.com/docker/hub-feedback/issues/552).